ext/nokogiri/extconf.rb

```diff
@@ -563,18 +580,19 @@ def install
   "exslt" => ['exsltFuncRegister',      'libexslt/exslt.h'],
 }.each { |lib, (func, header)|
   have_func(func, header) ||
-  have_library(lib, func, header) ||
+  have_library(lib, func, header, iconv_libs) ||
```

Comment:
> If that's all you need, passing --with-opt-lib=/opt/local/lib will do, I think.

Is this when using `--use-system-libraries`?

`have_library` works fine then and yes I assume it is enough. When building `libxml-2.0` and `libxslt` it knows nothing and errors as per [this](https://github.com/sparklemotion/nokogiri/issues/1235#issuecomment-71496544).